test/nodes/test_retriever.py

```diff
@@ -444,6 +444,36 @@ def test_table_text_retriever_embedding(document_store, retriever, docs):
         assert isclose(doc.embedding[0], expected_value, rel_tol=0.001)
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize("retriever", ["table_text_retriever"], indirect=True)
+@pytest.mark.parametrize("document_store", ["memory"], indirect=True)
+@pytest.mark.embedding_dim(512)
+def test_table_text_retriever_embedding_only_table(document_store, retriever):
```

Comment:
This should be renamed:
```suggestion
def test_table_text_retriever_embedding_only_text(document_store, retriever):
```